README.md

```diff
@@ -217,6 +212,11 @@ Install uv:
 curl -LsSf https://astral.sh/uv/install.sh | sh
 ```
 
+Install Python 3.10, if it is not already installed:
+```shell
+uv python install 3.10
```

Comment:
This does not need to be done explicitly as long as the `uv sync` workflow is being used. uv will automatically download and install an appropriate python when it creates the virtualenv.

Reply:
Huh! Good to know. I'll merge this PR, and remove this change in the next PR.